CNAME

```diff
@@ -1 +0,0 @@
-read.wiki.org
```

Comment:
It makes sense that the `CNAME` is only needed on the `gh-pages` branch. Presently, though, our deployment strategy has just been for `gh-pages` to track `master`.

I expect we'll move to a real build process eventually. At that point, I'd like a `publish` that'll generate the necessary stuff on the `gh-pages` branch. Until we get there, though, it's convenient to keep the file here... what do you think? Am I missing something there?